lib/board.rb

```diff
@@ -0,0 +1,61 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells, :board
+
+  def initialize
+    # @board= @cells
+    @cells= Array.new(9, " ")
+  end
+
+  def reset!
+    @cells= Array.new(9, " ")
+  end
+
+  def display
+    puts "\n"
+    puts "\n"
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+    puts "\n"
+    puts "\n"
+  end
+
+  def position(input)
+    @cells[input.to_i - 1 ]
+  end
+
+  def full?
+     @cells.include?(" " ||"") ? false : true
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. 

`!@cells.include?(" " || "")`